RFC: TOC can stand in for WG leads

ameowlia opened the pull request:
🚀 [Link for easy viewing.](https://github.com/cloudfoundry/community/blob/toc-stand-in-for-wg-lead-rfc/toc/rfc/rfc-draft-toc-stand-in-for-wg-lead.md)

ChrisMcGowan commented:
Entering final comment period ending on 9/12/2023

Gerg commented:
I broadly support this RFC; we already have some prior art for TOC overrides ([example](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md#promotion-and-revocation)).

Operationally, I curious how "absence" will be established. How will you know that a WG Lead is absent? How long a period of absence is sufficient for TOC override?

You _could_ grant the TOC the ability to declare a WG Lead absent. At that point, we could simplify the process and let the TOC fill in for a WG Lead for any reason, at TOC discretion, without introducing the concept of "absence".

beyhan commented:
>At that point, we could simplify the process and let the TOC fill in for a WG Lead for any reason, at TOC discretion, without introducing the concept of "absence".

Good point. +1 for having this as simpler as possible.

ameowlia commented:
> I broadly support this RFC; we already have some prior art for TOC overrides ([example](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md#promotion-and-revocation)).
> 
> Operationally, I curious how "absence" will be established. How will you know that a WG Lead is absent? How long a period of absence is sufficient for TOC override?
> 
> You _could_ grant the TOC the ability to declare a WG Lead absent. At that point, we could simplify the process and let the TOC fill in for a WG Lead for any reason, at TOC discretion, without introducing the concept of "absence".

Good idea. I updated the RFC to reflect this. I left the idea of "absence" in the problem statement to describe how this RFC came to be, but I removed "absence" as a requirement for when the TOC can step in.

ameowlia commented on toc/rfc/rfc-draft-toc-stand-in-for-wg-lead.md:
```diff
@@ -0,0 +1,25 @@
+## Summary
+
+Working Group Leads are in charge of approving and merging PRs for their
+Working Group. When the TOC decideds it is necessary, the TOC MAY make decisions in their stead.
```
```suggestion
Working Group. When the TOC decideds it is necessary, the TOC MAY make decisions in their place.
```

ameowlia commented on toc/rfc/rfc-draft-toc-stand-in-for-wg-lead.md:
```diff
@@ -0,0 +1,25 @@
+## Problem
+
+Sometimes Working Group Leads go on vacation or get sick and are not available
+to review PRs. This can cause progress on their teams to grind to halt until
+the Working Group lead returns.
```
```suggestion
to take actions for their working group. This can cause work on their working groups to be blocked until
the Working Group Lead returns.
```

ameowlia commented on toc/rfc/rfc-draft-toc-stand-in-for-wg-lead.md:
```diff
@@ -0,0 +1,25 @@
+
+## Proposal
+
+When the TOC decideds it is necessary, the TOC MAY make decisions in their stead. This
```
```suggestion
When the TOC decides it is necessary, the TOC MAY make decisions in their stead. This
```
